MvvmCross/Platforms/Ios/Presenters/MvxIosViewPresenter.cs

```diff
@@ -370,28 +363,23 @@ protected virtual bool CloseTabViewController(IMvxViewModel viewModel, MvxTabPre
             return false;
         }
 
-        protected virtual bool CloseModalViewController(IMvxViewModel viewModel, MvxModalPresentationAttribute attribute)
+        protected virtual bool CloseMasterSplitViewController(IMvxViewModel viewModel, MvxSplitViewPresentationAttribute attribute)
         {
-            return CloseModalViewController(viewModel);
-        }
-
-        protected virtual bool CloseMasterSplitViewController(IMvxViewModel viewModel, MvxMasterSplitViewPresentationAttribute attribute)
-        {
-            if (SplitViewController != null && SplitViewController.CloseChildViewModel(viewModel))
+            if (SplitViewController != null && SplitViewController.CloseChildViewModel(viewModel, attribute))
                 return true;
 
             return true;
         }
 
-        protected virtual bool CloseDetailSplitViewController(IMvxViewModel viewModel, MvxDetailSplitViewPresentationAttribute attribute)
+        protected virtual bool CloseDetailSplitViewController(IMvxViewModel viewModel, MvxSplitViewPresentationAttribute attribute)
         {
-            if (SplitViewController != null && SplitViewController.CloseChildViewModel(viewModel))
+            if (SplitViewController != null && SplitViewController.CloseChildViewModel(viewModel, attribute))
                 return true;
 
             return true;
         }
 
-        protected virtual bool CloseModalViewController(IMvxViewModel toClose)
+        protected virtual bool CloseModalViewController(IMvxViewModel toClose, MvxModalPresentationAttribute attribute)
```

Comment:
Many users will hit this breaking change, just leaving a note as a reminder for docs